toolkit/tools/imager/imager.go

```diff
@@ -481,6 +481,12 @@ func buildImage(mountPointMap, mountPointToFsTypeMap, mountPointToMountArgsMap m
 		setupChrootPackages = append(setupChrootPackages, toolingPackage.Name)
 	}
 
+	logger.Log.Infof("HidepidDisabled is %v.", systemConfig.HidepidDisable)
+	hidepidEnable := true
```

Comment:
Nit: to make it shorter you can also do `hidepidEnable := !systemConfig.HidepidDisable`.

Reply:
Done